static/js/containers/VideoDetailPage_test.js

```diff
@@ -107,70 +98,21 @@ describe('VideoDetailPage', () => {
     );
   });
 
-  it('opens the dialog when the edit button is clicked and video is editable', async () => {
-    let wrapper = await renderPage({editable: true});
-    assert.isFalse(wrapper.find(".edit").props().disabled);
-    wrapper.find(".edit").props().onClick();
-    assert.isTrue(store.getState().videoDetailUi.editDialog.visible);
-  });
-
-  it('edit button has no effect if video is not editable', async () => {
+  it('includes the share button and dialog', async () => {
     let wrapper = await renderPage();
-    assert.isTrue(wrapper.find(".edit").props().disabled);
-  });
-
-  it('edits the title and description and submits the data to trigger an update', async () => {
-    let wrapper = await renderPage({editable: true});
-
-    const newTitle = 'new title';
-    const newDescription = "omg this is the BEST VIDEO";
-    const expectedVideo = {
-      ...video,
-      title: newTitle,
-    };
-    updateVideoStub.returns(Promise.resolve(expectedVideo));
-    await listenForActions([
-      actions.videos.patch.requestType,
-      actions.videos.patch.successType,
-      SET_EDIT_DIALOG_VISIBILITY,
-      SET_DESCRIPTION,
-      SET_DESCRIPTION,
-      SET_TITLE,
-      SET_TITLE,
-      CLEAR_EDIT_DIALOG,
-    ], () => {
-      wrapper.find(".edit").props().onClick();
-      wrapper.find("#video-title").props().onChange({target: {value: newTitle}});
-      wrapper.find("#video-description").props().onChange({target: {value: newDescription}});
-      wrapper.find('Dialog').at(0).props().onAccept();
-    });
-
-    assert.deepEqual(store.getState().videos.data.get(expectedVideo.key), expectedVideo);
-    sinon.assert.calledWith(updateVideoStub, video.key, {
-      title: newTitle,
-      description: newDescription,
-    });
+    assert.isTrue(wrapper.find(".share").exists());
+    assert.isTrue(wrapper.find("ShareVideoDialog").exists());
   });
 
-  it('show the share dialog', async () => {
```

Comment:
Bring back share dialog unit tests?

Reply:
The test for the content of the share dialog was replicated here: https://github.com/mitodl/odl-video-service/pull/224/files/bd4e940252a02b82ec680501abaf1b8c01fb97b7#diff-00c8a362f3a08ddcb7355a517c8f9d7eR42. The tests for showing and hiding a simple dialog are now
in [hoc_test.js](https://github.com/mitodl/odl-video-service/pull/224/files/bd4e940252a02b82ec680501abaf1b8c01fb97b7#diff-cd5b8f4619ea92fc1bf81505be1a1438).